docs/intro-to-dashboards-and-apps/connecting-etl-to-dashboard-with-voila.md

```diff
@@ -6,9 +6,178 @@ jupyter:
+
+```{important}
+During this process, it is essential to become acquainted with the various types of data present in each table, such as numerical, categorical, boolean, and others, before proceeding with data visualization. This familiarity aids the user in developing an intuition about the most suitable visualizations for emphasizing columns individually or exploring relationships between them.
+```
+
+### Questions Answered through Visualizations
+
+After the user interacts with the tables, questions that the dashboard answers with the help of visualizations are then listed (see above). It is recommended to begin with simpler, more high-level questions about the data, such as addressing counts of different vehicle types or distributions of emissions, before diving into relationships between columns. This allows the user to make more informed decisions about the types of visualizations to use for answering more complex questions.
+
+### Interactive Visualizations
+
+#### `Seaborn` Bar Plots
+
+The first visualization is a `barplot()`, built with `seaborn` and `ipywidgets`, of the number of unique fuel-only, electric, and hybrid car models, visualized across the years they were released. A trend overlay line (`pointplot()`) is added to clearly spot yearly manufacturing patterns in these vehicles. Interactivity is enabled through `RadioButtons`, which helps the user toggle between a bar plot of either fuel-only cars or a column chart of hybrid and electric cars. The decision to provide this separation is because fuel-only cars' models date back to 1995 in the dataset, while hybrid and electric cars' models date back to 2012. Therefore, the previous recommendation that the user should get familiar with the data before visualizing it is important here, which also assists in thinking about both the type of widget and where it could be utilized with respect to the data.
+
+Lastly, notice that <b>two CTE's</b>, one that uses the `fuel` table for the fuel-only bar plot and another that uses the `all_vehicles` table for the electric and hybrid bar plot, were created and converted into `pandas` DataFrames to pass into our `seaborn` function that initializes the plots. From `ipywidgets`, the [`interact` function](https://ipywidgets.readthedocs.io/en/latest/examples/Using%20Interact.html), which takes as input the `seaborn` function and its parameter, is then called to visualize the plots with the `RadioButtons`. A basic example of the `interact` function is shown below:
```

Comment:
It's hard to see what the CTE's referring to without seeing it rendered in the blog

Reply:
Added both CTEs for bar plot in latest commit